src/OrdinaryDiffEq.jl

```diff
@@ -134,9 +135,7 @@ module OrdinaryDiffEq
   export ode_addsteps!, ode_interpolant,
         terminate!, savevalues!, copyat_or_push!, isfsal
 
-  export constructDP5, constructVern6, constructDP8,
-         constructDormandPrince, constructFeagin10,
-         constructFeagin12, constructFeagin14
```

Comment:
FYI there is `Feagin10ConstantCache`. Maybe you want to export `Feagin10ConstantCache`?

https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/47e9711877b3a6767364623fd0af8d404b68488c/src/tableaus/feagin_tableaus.jl#L269-L273

Reply:
Nope, those tableaus shouldn't be exported. This should just be removed. The tableaus that should be exported are now in DiffEqDevTools

Reply:
I see.